src/param/src/lib.rs

```diff
@@ -166,7 +166,7 @@ impl Param {
             content_id: content_id.ok_or(ReadError::MissingContentId)?,
             title,
             title_id: title_id.ok_or(ReadError::MissingTitleId)?,
-            version: version.ok_or(ReadError::MissingVersion)?,
```

Comment:
This need to be handled on the UI side. What you need to do here is make version as an optional.

Reply:
Not sure how that's UI sided when it... Gives the error

```rs
    #[error("VERSION parameter not found")]
    MissingVersion,
 ```
 
 ?
 
 Which also means even if ignored in UI, it'd crash on the kernel... When obviously the PS4 can run its own UI without the version param.